cmd/api/notifications/notifications_test.go

```diff
@@ -0,0 +1,176 @@
+package notifications_test
+
+import (
+	"bufio"
+	"context"
+	"errors"
+	"fmt"
+	"io"
+	"net/http"
+	"net/http/httptest"
+	"strings"
+	"testing"
+	"time"
+
+	"github.com/books-service/cmd/api/book"
+	"github.com/books-service/cmd/api/notifications"
+	notificationmocks "github.com/books-service/cmd/api/notifications/mocks"
+	"github.com/google/uuid"
+	"github.com/matryer/is"
+	"go.uber.org/mock/gomock"
+)
+
+// Integration tests:
```

Comment:
You already have an indicator of the test type in the name of the test function, I wouldn't add the comment, its pretty clear without it.